testdata/mongodb/testCollections.js

```diff
@@ -0,0 +1,39 @@
+for (let i = 1; i <= 5; i++) {
+  db = db.getSiblingDB(`testingCollections${i}`);
+  db.customers.insert({ name: 'Honey1', age: 25, cars: ['Audi R8'] });
+  db.customers.insertMany(
+    [
+      {
```

Comment:
can we extract this object in constant(it's used 3 times)?

Reply:
no, I don't know how to do that and it kills the readability

Reply:
I thought the readability will be improved because it will be less data in loop
```
const dbObjects = [
      {
        name: 'Midhuna2', age: 23, cars: ['BMW 320d', 'Audi R8'], place: 'Amaravati',
      },
      {
        name: 'Akhil2', age: 24, cars: ['Audo A7', 'Agera R'], place: 'New York',
      },
      { name: 'Honey2', age: 25, cars: ['Audi R8'] },
    ];

db.products.insertMany(dbObjects);
```

Reply:
@nazarovvi please look here and handle this ^

Reply:
Fixed the code duplication